src/CLI/renderer/templates/show_tam_collector.j2

```diff
@@ -0,0 +1,33 @@
+{% set vars = {'name': ""} %}
+{% set vars = {'ipaddresstype': ""} %}
+{% set vars = {'address': ""} %}
+{% set vars = {'port': ""} %}
+{% if json_output -%}
+{% for key_json in json_output %}
+    {% if key_json == "sonic-ifa:TAM_COLLECTOR_TABLE" %}
+        {% set collector_list = json_output[key_json]["TAM_COLLECTOR_TABLE_LIST"] %}
+    {% else %}
+        {% set collector_list = json_output[key_json] %}
+    {% endif %}
+{{'---------------------------------------------------'}}
+{{'TAM Collector Information'.ljust(20)}}
+{% for collector in collector_list %}
+    {% if vars.update({'name':collector["name"]}) %}{% endif %}
+    {% if vars.update({'ipaddresstype':collector["ipaddress-type"]}) %}{% endif %}
+    {% if vars.update({'port':collector["port"]}) %}{% endif %}
+    {% if vars.update({'address':collector["ipaddress"]}) %}{% endif %}
+---------------------------------------------------
+Collector                : {{ vars.name }}
```

Comment:
@spenugondaa  :The  show commands should be displayed in tabular format as shown below:

oot@sonic:/home/admin#  show tam collector all

NAME         IP TYPE          IP                 PORT

C1                ipv4          10.10.10.10      9070

C2                ipv4          20.20.20.20      9071

Reply:
Done